src/main/java/com/teragrep/pth_06/scheduler/ArchiveBatchSliceCollection.java

```diff
@@ -49,8 +49,10 @@
 import com.teragrep.pth_06.planner.ArchiveQuery;
 import com.teragrep.pth_06.planner.offset.DatasourceOffset;
 import org.apache.spark.sql.connector.read.streaming.Offset;
+import org.jetbrains.annotations.NotNull;
```

Comment:
remove unused import

Reply:
removed unused imports in ArchiveBatchSliceCollection